requirements-dev.txt

```diff
@@ -1,5 +1,4 @@
 # Data import
-pandas==0.16.2
 xlrd==0.9.4
```

Comment:
Should we remove this from in here as well since it's in the main `requirements.txt` file?